From: Hannah von Reth <vonreth@kde.org>
Date: Mon, 18 Mar 2013 14:34:41 +0000
Subject: disable the audio cd collection on windows, it would build but not work
X-Git-Url: http://quickgit.kde.org/?p=amarok.git&a=commitdiff&h=a7d330ed8e78374c3117ca0c292916d269a3dd15
---
disable the audio cd collection on windows, it would build but not work
---


--- a/src/core-impl/collections/CMakeLists.txt
+++ b/src/core-impl/collections/CMakeLists.txt
@@ -7,7 +7,10 @@
 set(AMAROK_COLLECTION_SUPPORT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/support)
 
 add_subdirectory( daap )
+
+if( NOT WIN32)
 add_subdirectory( audiocd )
+endif( NOT WIN32 )
 
 add_subdirectory( ipodcollection )
 add_subdirectory( mtpcollection )
